native/android/app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,6 @@
 <resources>
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <item name="android:windowExitAnimation">@android:anim/fade_out</item>
-        <item name="android:textColor">#000000</item>
```

Comment:
This should not be removed, otherwise we might have white color on white background again.